ghedesigner/utilities.py

```diff
@@ -285,3 +285,59 @@ def write_flat_dict_to_csv(write_path: Path, input_dict: dict) -> None:
 
         for row in zip(*input_dict.values()):
             writer.writerow(row)
+
+
+def read_csv_column(file_path: str | Path, column: int | str, try_convert_to_numeric=True) -> list:
+    """
+    Reads a specific column from a CSV file.
+
+    Parameters:
+        file_path (str or Path): Path to the CSV file.
+        column (str or int): Column name (str) or index (int) to extract.
+        try_convert_to_numeric (bool, optional): If True, convert column to numeric. Defaults to True.
+
+    Returns:
+        list: A list of values from the specified column.
+    """
+    values = []
+
+    with open(file_path, newline="", encoding="utf-8") as csv_file:
+        reader = csv.reader(csv_file)
+        header = next(reader)
+
+        # Determine column index
+        if isinstance(column, str):
+            try:
+                col_index = header.index(column)
```

Comment:
I suppose a duplicate column name would just give the first instance.  This is fine, and likely not a problem, just poking around...